jdaviz/cli.py

```diff
@@ -100,7 +121,7 @@ def _main(config=None):
     parser.add_argument('--instrument', type=str, default='nirspec',
                         help='Manually specifies which instrument parser to use, for Mosviz')
     parser.add_argument('--browser', type=str, default='default',
-                        help='Browser to use for application.')
+                        help='Browser to use for application (use qt for embedded qt browser).')
```

Comment:
What is an "embedded qt browser"? If you require it for standalone to work, then is Jdaviz turning into Qt app (again)?

Reply:
With embedded I try to communicate that it will not use the system browser, suggestions are welcome.

Not sure I understand what you mean by the second sentence exactly, but it's not a Qt app.

Reply:
Yes, it's using the Qt browser (chromiums based I think), but that could be any embeddable browser. We are not using any of the Qt widgets for the UI.